internal/provider/resource_consul_cluster.go

```diff
@@ -69,7 +69,7 @@ func resourceConsulCluster() *schema.Resource {
 			},
 			"tier": {
 				// TODO: link to HCP Consul feature tier page when it is available
-				Description:      "The tier that the HCP Consul cluster will be provisioned as.  Only `development` and `standard` are available at this time.",
+				Description:      "The tier that the HCP Consul cluster will be provisioned as.  Only `development`, `standard` and `plus` are available at this time.",
```

Comment:
Looks like the pricing page exists now if you want to add the link here. 
```suggestion
				Description:      "The tier that the HCP Consul cluster will be provisioned as.  Only `development`, `standard` and `plus` are available at this time. See [pricing information](https://cloud.hashicorp.com/pricing/consul).",
```